projects/plugins/boost/app/assets/src/js/pages/settings/sections/Modules.svelte

```diff
@@ -54,11 +54,8 @@
 			/>
 		</p>
 
-		<div slot="meta">
-			<button type="button" class="components-button is-link" on:click={requestCloudCss}>
-				<RefreshIcon />
-				{__( 'Regenerate', 'jetpack-boost' )}
-			</button>
+		<div slot="meta" class="jb-feature-toggle__meta">
+			<GenerateCloudCss />
```

Comment:
I think the name `GenerateCloudCss` is a bit misleading. May be this should be something like `CloudCSSMeta`?